README.rst

```diff
@@ -8,7 +8,18 @@ MantisShrimp
 
 --------------
 
-``MantisShrimp`` provides a unified data API for object detection that can be used with any library (it can even be injected in code that was not supposed to work as a library, take a look at ``hub/detr`` for an example).
+.. image:: images/mantisshrimp-logo.png
+
+Why Mantishrimp
+---------------
+- Mantisshrimp: An object-detection library
+- Built on top of different libraries/framework such as Pytorch Lightning and Pytorch
+- Features a Unified Data API such: common Parsers (COCO, etc.),
+- User-defined Parsers (e.g. WheatParser)
```

Comment:
Can we say: "Community maintaned parsers for custom datasets"? Or something like that

Reply:
Maybe:
Integrates community maintaned parsers for custom datasets shared on mantisshrimp parsers hub

Reply:
I like that!

Only suggestion: "mantisshrimp parsers hub" -> "parsers hub"